packages/js/components/src/confetti-animation/index.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import confetti from 'canvas-confetti';
+import { useEffect } from '@wordpress/element';
+
+// Original source: https://github.com/Automattic/wp-calypso/blob/a39539547780871d0371a20fcf21c767a86a1010/packages/components/src/confetti/index.ts
```

Comment:
I couldn't import from `@automattic/components` since it's not included in the currently available version and updating the package requires updating a bunch of others with breaking changes.